### DISCUSSION TITLE
White-label Open WebUI

### DISCUSSION THREAD

**Discussion by rulhaq** [open-webui/open-webui#5791] [Q&A]
# White-labeling is supported via https://docs.openwebui.com/enterprise/

**Comment by Killamyflash** [comment]
guys ... wtf ur customer ?? u sell a open source system for money to your customers?? go and create your own web interface...  what a hell is going on with u ppl.

**Reply by rulhaq** [reply]
agreed, as per license terms, if you plan to sell opensource, you dont. You can charge for your PS and the time you will invest in consulting and deploying, as in man hours, but never and i mean NEVER for software. Let me know if you need to learn to structure and strategize your sales around opensource solutions. AND if you make any changes to the interface of Open-WebUI, you need to inform the creator and give it back to community as well. otherwise, build your own!

**Reply by Killamyflash** [reply]
i dont understand this... this guys do a great job and didnt earn any money for this... they should get respected for the work. and if anyone wants a own Brandig then they should pay for it monthly!

**Comment by Killamyflash** [comment]
we created wordpress plugin for openwebui to connect our website to the API and integrate TTS ... i wanted to release it next week 4 free. but after i read the posts here... i am thinkig about to make a payment plan 4 it and spend all the money to the dev team of Openwebui .... really i hate it when ppl so ********

**Reply by rulhaq** [reply]
Freemium is the way to go! All the best for the release mate.
